templates/items/parts/item-advancement2.hbs

```diff
@@ -0,0 +1,112 @@
+<div class="tab advancement" data-group="primary" data-tab="advancement">
+    <section class="items-list">
+        {{#each advancement}}
+
+        <div class="items-section card" data-level="{{ @key }}">
```

Comment:
Modify choices isn't working at the moment because it is looking for `<li>` to find the level. You'll have to modify line `814` of `item-sheet.mjs` to look for the data instead:

```javascript
const level = target.closest("[data-level]").dataset.level;
```